spec/support/acceptance/splunk/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "2.0"
+
+services:
+  enterprise:
+    image: splunk/splunk:latest
+    hostname: splunk_instance
+    environment:
+      - SPLUNK_START_ARGS=--accept-license
+      - ASFDGASDFG=5
+      # The splunkbase credentials are to download and install the Puppet
+      # Report Viewer from Splunkbase.
+      # We could alternatively download the packaged app from a location
+      # like github for testing.
+      - SPLUNK_APPS_URL=https://splunkbase.splunk.com/app/4413/release/3.0.1/download
+      # TODO: Make a PIE account
+      - SPLUNKBASE_USERNAME=gregsparks
+      - SPLUNKBASE_PASSWORD=2Riboflavin!
+      - SPLUNK_PASSWORD=piepiepie
+    volumes:
+      # default.yml is a mechanism to load splunk settings that would normally
+      # be configured through the ui.
+      - ./default.yml:/tmp/defaults/default.yml
+    ports:
+      # localhost:8000 will bring up the web interface
+      - "127.0.0.1:8000:8000"
+      # 8088 is the hec endpoint
+      - "127.0.0.1:8088:8088"
```

Comment:
Changing these bindings to `0.0.0.0`  might make things easier later for things like troubleshooting the tests and prevent an annoying bug for someone later on.